src/login/logind-action.c

```diff
@@ -18,6 +18,8 @@
 #include "terminal-util.h"
 #include "user-util.h"
 
+/* For INHIBIT_SHUTDOWN items in this array, 'log-verb' is sent out by PrepareForShutdownWithMetadata signals,
+ * so the values cannot change as they are public APIs. */
```

Comment:
that sounds wrong. instead, send `handle_action_to_string()` of the `.handle` field.